StylableSwiftUI/Classes/Logger.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Verbosity.swift
+//  StylableSwiftUI
+//
+//  Created by Kerr Marin Miller on 15/04/2020.
+//
+
+import Foundation
+
+public final class Logger {
+
+    public static let `default` = Logger()
+
+    public var level: Level
+
+    public enum Level: Int {
+        case debug = 0
+        case info
+        case warning
+        case error
+    }
+
+    public init() {
```

Comment:
Why is the init method public - do we ever want anyone outside this library creating one?